src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -68,6 +74,56 @@ struct ExcisionSphere : db::SimpleTag {
   }
 };
 
+/*!
+ * \brief The position of the scalar charge particle orbiting the worldtube
+ * given in inertial coordinates. We currently assume a circular orbit in the
+ * xy-plane with radius \f$R\f$ and angular velocity \f$\omega = R^{-3/2}\f$,
+ * where grid and inertial coordinates are equal at t = 0.
+ */
```

Comment:
Please add a brief note why this can't be done with maps at the moment. Something like coordinate maps are currently stored only in blocks, and the worldtube is an excision region outside any block. We could inject the time-dependent grid -> inertial map into the worldtube to fix this.

I think we can currently compute the center of a Strahlkorper and transform a Strahlkorper to another frame, but that seems too complicated.

IMO you could just add an optional time-dependent grid -> inertial map to ExcisionSphere, similar to how it's done in Block.

Reply:
yes, I was thinking we should inject the time dependent map into the excision sphere. Once Shell and Sphere are merged together it can just be the same map that would go into the block that would fill the excision sphere for all domain creators.